templater/jobs/presubmit/eks-anywhere-build-tooling/envoy-presubmit.yaml

```diff
@@ -0,0 +1,16 @@
+jobName: envoy-presubmit
+runIfChanged: ^build/lib/.*|Common.mk|projects/envoyproxy/envoy/.*
```

Comment:
Not on you, but we should check the projects that do not have a base image tag file in the runifchanged, like this one, I suspect most of them should